kernel/src/engine/arrow_expression.rs

```diff
@@ -62,7 +64,7 @@ impl Scalar {
                     PrimitiveType::String => Arc::new(StringArray::new_null(num_rows)),
                     PrimitiveType::Boolean => Arc::new(BooleanArray::new_null(num_rows)),
                     PrimitiveType::Timestamp | PrimitiveType::TimestampNtz => {
```

Comment:
Btw, this will create an array with timezone UTC for timestampNtz as well

Reply:
thanks! fixed it.